### ISSUE TITLE
[Feature request] Being Able to set a min and max layer height in the adaptative layering menu

### ISSUE THREAD

**Issue by NairoDorian** [bambulab/BambuStudio#901]
I was trying to use the latest implementation of the Adaptative Layer Menu, but unfortunately while trying to setup the layering for my parts I reached a layer height too small and the melted filament is somehow too close to the previous layer and causes problems.

A quick and very useful fix would be to be able to setup a  minimum and a maximum layer height value for the adaptative layer function.

**Comment by FlowForm** [comment]
Yes, this function is very needed. Very much like Cura does it.

**Comment by cameo69** [comment]
I second that.

**Comment by cann0nf0dder69** [comment]
No need to reinvent the wheel, it works well in cura, adopt what they're doing instead of this quality/speed/radius rubbish

**Comment by MadMax389** [comment]
You can already do this in the printer profile...
![image](https://user-images.githubusercontent.com/96731892/210121979-93a92b6c-6a3b-4986-8ddb-b302ad00f3bb.png)

**Comment by NairoDorian** [comment]
Yes I just discovered that feature, it does help to set a min & max for the adaptative layering indeed, but not per object
